Sources/Testing/Test+Macro.swift

```diff
@@ -484,6 +484,27 @@ extension Test {
   }
 }
 
+// MARK: - Test pragmas
+
+/// A macro used similarly to `#pragma` in C or `@_semantics` in the standard
+/// library.
+///
+/// - Parameters:
+///   - arguments: Zero or more context-specific arguments.
+///
+/// The use cases for this macro are subject to change over time as the needs of
+/// the testing library change. The implementation of this macro in the
+/// TestingMacros target determines how different arguments are handled.
+///
+/// - Note: This macro has compile-time effects _only_ and should not affect a
+///   compiled test target.
+///
+/// - Warning: This macro is used to implement the `@Test` macro. Do not use it
+///   directly.
+@attached(peer) public macro __testing(
```

Comment:
Does this _need_ to be `public` initially? I suspect that's the direction this is headed regardless, but just wondering if it's only needed by our own test code right now

Reply:
We can make it internal for now—there's a public version coming almost immediately after this PR though.

Reply:
See https://github.com/swiftlang/swift-testing/pull/745/commits/ddfdbc9a751f167dd4c6c99cb4c949739f813182